### PR TITLE
Added Playground for Android

### PR DESCRIPTION
**Pull request by imanishbarnwal**
Issue: #13

### CODE REVIEW

**Review by king-11 — CHANGES_REQUESTED** [review]
@imanishbarnwal there is an additional line break at the end which isn't part of your change I believe if possible please remove it as will result in it merge conflicts at a later point for other PRs.

**Comment by imanishbarnwal** [comment]
> @imanishbarnwal there is an additional line break at the end which isn't part of your change I believe if possible please remove it as will result in it merge conflicts at a later point for other PRs.

I checked but didn't get that. Can you please link the line here?

File: README.md [hunk 1]
```diff
@@ -93,4 +94,4 @@ Share your apps here! send a pull request!
 * [Twitter](https://twitter.com/appwrite_io) (Official, global)
 * [Facebook](https://www.facebook.com/appwrite.io/) (Official, global)
 * [Appwrite Developers on Facebook](https://www.facebook.com/groups/appwrite.developers/?source_id=227046654430085) (Official, global)
-* [Gitter](https://gitter.im/appwrite/community) (Legacy, global)
\ No newline at end of file
+* [Gitter](https://gitter.im/appwrite/community) (Legacy, global)
```
Comment (king-11) [inline README.md:97]: There is one line break character at it isn't visible as its just a \n try using a code editor like atom, sublime or vscode then you will see a blank line at end you can then remove that line

Comment (imanishbarnwal) [reply README.md:97]: Nope, still not visible. I think it's showing because I have added a line in between so last line is shifted below.

Comment (king-11) [reply README.md:97]: I tried cloning your repo and found it's there you just have to remove the line.

Comment (imanishbarnwal) [reply README.md]: yeah got it thanks, and also just updated the file. Now it's fine.

**Review by king-11 — APPROVED** [review]

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Comment by imanishbarnwal** [comment]
> THANK YOU! All changes merged 🥳
> 
> Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.
> 
> As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.
> 
> You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

Thanks, @eldadfux just joined the discord I will message you there. 😊

**Pull request merged**